src/norm/postgres.nim

```diff
@@ -98,6 +98,14 @@ proc createTables*[T: Model](dbConn; obj: T) =
       fkGroups.add "FOREIGN KEY($#) REFERENCES $#($#)" %
         [obj.col(fld), typeof(get val.model).table, typeof(get val.model).col("id")]
 
+    when obj.dot(fld).hasCustomPragma(fk):
+      const pragVal = $(obj.dot(fld).getCustomPragmaVal(fk))
```

Comment:
Could be one `const` block.

Reply:
I'm not comfortable with that string conversion. What is the point of that line? If you need the table name, use `table`.

Reply:
table is better than string conversion indeed.